Makefile: do not build tests

Tests are broken if no EGL support is available.

Reported-by: Peter Korsgaard <jacmet@uclibc.org>
Signed-off-by: "Yann E. MORIN" <yann.morin.1998@free.fr>
diff -durN weston-1.3.0.orig/Makefile.am weston-1.3.0/Makefile.am
--- weston-1.3.0.orig/Makefile.am	2013-08-20 20:15:19.000000000 +0200
+++ weston-1.3.0/Makefile.am	2013-10-14 23:08:12.016392097 +0200
@@ -2,7 +2,7 @@
 wcap_subdir = wcap
 endif
 
-SUBDIRS = shared src clients data protocol tests $(wcap_subdir) man
+SUBDIRS = shared src clients data protocol $(wcap_subdir) man
 
 DISTCHECK_CONFIGURE_FLAGS = --disable-setuid-install
 
